test/integration/supervisor_warnings_test.go

```diff
@@ -232,8 +232,8 @@ func TestSupervisorWarnings_Browser(t *testing.T) {
 			requireKubectlGetNamespaceOutput(t, env, string(kubectlPtyOutputBytes2))
 		}
 		// the output should include a warning that the groups have changed.
-		require.Contains(t, string(kubectlPtyOutputBytes2), fmt.Sprintf("User %q has been added to the following groups: %q", env.SupervisorUpstreamLDAP.TestUserMailAttributeValue, env.SupervisorUpstreamLDAP.TestUserDirectGroupsDNs))
-		require.Contains(t, string(kubectlPtyOutputBytes2), fmt.Sprintf("User %q has been removed from the following groups: [\"some-other-group\" \"some-wrong-group\"]", env.SupervisorUpstreamLDAP.TestUserMailAttributeValue))
+		require.Contains(t, string(kubectlPtyOutputBytes2), fmt.Sprintf("Warning:\x1b[0m"+` User %q has been added to the following groups: %q`+"\r\n", env.SupervisorUpstreamLDAP.TestUserMailAttributeValue, env.SupervisorUpstreamLDAP.TestUserDirectGroupsDNs))
```

Comment:
`x1b[0m` has something to do with the terminal output color formatting, right? Can you extract it as a variable to interpolate into the string so it's more clear what it's doing there?

Reply:
Done.